test/e2e/framework/context.go

```
@@ -240,3 +248,30 @@ func (th *TestContext) dumpControllerLogs() {
 		th.Logf("=== Controller logs (last 42 lines) from %s (full logs: %s) ===\n%s", pod.Name, logFile, tailBuf.String())
 	}
 }
+
+// ReportMetric will report a benchmark result.
+// If running as a benchmark it will be reported through the benchmark framework.
+// If running as a unit-test it will be logged through the test.
+func (th *TestContext) ReportMetric(n float64, unit string) {
+	if th.benchmark != nil {
+		th.benchmark.ReportMetric(n, unit)
+	} else {
+		th.T.Logf("benchmark[%v]=%v", unit, n)
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "T" from selector

(staticcheck)